src/NuGetForUnity/Editor/NugetConfigFile.cs

```diff
@@ -74,6 +74,20 @@ public class NugetConfigFile
         /// </summary>
         public bool ReadOnlyPackageFiles { get; set; }
 
+        public string PackagesConfigPath { get; set; }
```

Comment:
```suggestion
        public string PackagesConfigDirectoryPath { get; set; }
```
I like to be explicit even if the name is long